cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -0,0 +1,283 @@
+import sys
+import json
+import argparse
+from os import linesep as LINESEP
+
+# backward compatibility with 2.x
+try:
+    ModuleNotFoundError
+except:
+    ModuleNotFoundError = ImportError
+
+try:
+    import rados
+except ModuleNotFoundError as noModErr:
+    print("Error: %s\nExiting the script..." % noModErr)
+    sys.exit(1)
+
+try:
+    # for 2.7.x
+    from StringIO import StringIO
+except ModuleNotFoundError:
+    # for 3.x
+    from io import StringIO
+
+
+class ExecutionFailureException(Exception):
+    pass
+
+
+class RadosJSON:
+    EXTERNAL_USER_NAME = "client.healthchecker"
+
+    @classmethod
+    def gen_arg_parser(cls):
+        argP = argparse.ArgumentParser()
+        argP.add_argument("--conf", "-c",
+                          help="Provide a ceph conf file.", type=str)
+        argP.add_argument("--run-as-user", "-u",
+                          help="Provides a user name to check the cluster's health status",
+                          default=cls.EXTERNAL_USER_NAME, type=str)
+        argP.add_argument("--json", "-j", action="store_true",
```

Comment:
What about using `--format` and then it will either take `json` or `plain` as parameter value?

Reply:
Done